### DISCUSSION TITLE
RDFLib V7 Release

### DISCUSSION THREAD

**Discussion by aucampia** [RDFLib/rdflib#2501] [Development]
There have not been that many changes since the last release, but there are some fixes that affect work by @floresbakker and other critical fixes relating to quad handling, and the state of main is always really ready to release, so I think we can make a release if there are no objections.

The one downside of releasing now is that it is likely the next version will be version 8, but that is the case anyway, as there are many more breaking changes that should be made and not enough people around to make them.

If we wait longer, the question would be: What will we be waiting for? I will have some time a bit later in the summer to work on some changes that improve literals, which we need to further improve quad handling, but I'm uncertain if that is worth waiting for.

If there are no objections to releasing V7 from main more or less as is with maybe one or two small PRs incorporated, then I will make a release this weekend (28th and 29th, i.e before 2023-07-30).

CC: @RDFLib/core @RDFLib/core-reviewers

**Comment by aucampia** [comment]
RElease notes will look something like this: 

https://gist.github.com/aucampia/a2d4109f8d38307825295df23d6961d1

**Comment by floresbakker** [comment]
Sounds good :) but I am obviously positively biased towards an early release. Will the formal release page for rdflib also be updated?

**Comment by white-gecko** [comment]
I don't have any objections towards a release.

**Comment by aucampia** [comment]
Apologies, I was a bit preoccupied this weekend. I will start work on the release tonight (CET), there are a couple of PRs to merge first but it should not take too much work.

**Comment by floresbakker** [comment]
No apologies needed. It would be awesome if the release could be done within three weeks. So releasing tonight or this week is super nice but do not worry if you run into a little delay.

**Comment by aucampia** [comment]
All PRs that should go into the release are now merged, I will write release notes and do the rest of the release prep tonight.

**Comment by aucampia** [comment]
Released now https://github.com/RDFLib/rdflib/discussions/2513

**Reply by floresbakker** [reply]
Awesome! Thank you @aucampia !
